### ISSUE TITLE
No exclude for folders

### ISSUE THREAD

**Issue by cajoy** [MatthieuLemoine/unused-webpack-plugin#5]
I've tried to exclude folder like `'containers/template/**/*.*'`

but it doesn't work

`containers/template/*` doesn't work either

**Comment by MatthieuLemoine** [comment]
Yeah, actually it's linked to a library I use that doesn't support excluding directories. See https://github.com/jergason/recursive-readdir/issues/48

**Comment by MatthieuLemoine** [comment]
recursive-readdir can be passed an ignore function see https://github.com/jergason/recursive-readdir/blame/master/README.md#L42.

So you should be able to pass a function instead.

Could be good to have built-in support for excluding directories.

Feel free to contribute !

**Comment by AlahmadiQ8** [comment]
I would like to claim this and work on adding this feature.

**Comment by MatthieuLemoine** [comment]
Fixed in [v2.2.0](https://github.com/MatthieuLemoine/unused-webpack-plugin/releases/tag/v2.2.0)
